### PR TITLE
Test/e2e first exam 

### PR DESCRIPTION
**Pull request by utku-guclu**
poäng equality test for the first completed exam in the courses section.
[ticket](https://trello.com/c/OPFSozYv/1005-e2e-test-prognos-exam)

### CODE REVIEW

File: .github/workflows/e2e-tests.yml
Comment (lucasgojeda) [inline .github/workflows/e2e-tests.yml:1]: Please remove this file, we are no longer going to run the github action for now

File: e2e/tests/Poäng/poäng.spec.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+import { baseFixture } from "../../fixtures";
+
+/** Data */
+import * as testCredentials from "../../data/test_user_credentials.json";
+
+/** Pages */
+import { ExamSection } from "../../pages";
+
+test.describe("Test poäng number equality", () => {
+  test.fixme(
```
Comment (lucasgojeda) [inline e2e/tests/Poäng/poäng.spec.js:14]: Remove this functionality, we should run this test for the 3 browsers if it is possible, if there are failures that are not related to the logic of the code, then we can dismiss it but if it's possible we should run the 3

**Comment by lucasgojeda** [comment]
Just two comments, the rest of it looks great!

**Comment by lucasgojeda** [comment]
Good job Utku!

**Review by lucasgojeda — APPROVED** [review]

**Comment by utku-guclu** [comment]
Thank you Lucas!

**Pull request merged**